### PR TITLE
Update the changelog all the way to rc1.

### PR DESCRIPTION
**Pull request by vpetrovykh**

### CODE REVIEW

File: docs/changelog/5_x.rst [hunk 1]
```diff
@@ -440,3 +440,102 @@ Bug fixes
...
+* Separate triggers by kind in compilation.
+  (:eql:gh:`#7082`)
+
+* Set "Connection: close" for non-keep-alive requests
+  (:eql:gh:`#7087`)
+
+* Fix fts::search volatility
+  (:eql:gh:`#7106`)
+
+* Allow trailing separator for ListNonterm
```
Comment (aljazerzen) [inline docs/changelog/5_x.rst:469]:
```suggestion
* Allow trailing commas and semicolons in most places
```

File: docs/changelog/5_x.rst [hunk 2]
```diff
@@ -440,3 +440,102 @@ Bug fixes
...
+
+* Generalize some aspects of handling object-only indexes
+  (:eql:gh:`#7179`)
+
+Bug fixes
+---------
+
+* Add rudimentary support for inline function calls
+  (:eql:gh:`#7173`)
+
```
Comment (aljazerzen) [inline docs/changelog/5_x.rst:530]: This is an internal change

**Review by aljazerzen — APPROVED** [review]

File: docs/changelog/5_x.rst [hunk 3]
```diff
@@ -440,3 +440,102 @@ Bug fixes
...
   (:eql:gh:`#7073`)
+
+
+Beta 3
+======
+
+Additional changes
+------------------
+
+* Add ``cfg::Config._query_cache_mode``
```
Comment (msullivan) [inline docs/changelog/5_x.rst:451]: s/_query_cache_mode/query_cache_mode/

File: docs/changelog/5_x.rst [hunk 4]
```diff
@@ -440,3 +440,102 @@ Bug fixes
...
+  (:eql:gh:`#7158`)
+
+Bug fixes
+---------
+
+* Make escaping strings more consistent.
+  (:eql:gh:`#7059`)
+
+* Separate triggers by kind in compilation.
+  (:eql:gh:`#7082`)
```
Comment (msullivan) [inline docs/changelog/5_x.rst:461]: "Allow an update to trigger an insert of the same type, and vice versa"

File: docs/changelog/5_x.rst [hunk 5]
```diff
@@ -440,3 +440,102 @@ Bug fixes
...
+
+
+RC 1
+====
+
+EdgeDB + AI
+-----------
+
+We've added an ``ext::ai`` extension for handling integration of EdgeDB with
+various AI backends such as: OpenAI, Mistral and Anthropic.
```
Comment (beerose) [inline docs/changelog/5_x.rst:498]:
```suggestion
We've added an ``ext::ai`` extension for handling the integration of EdgeDB with
various AI backends such as: OpenAI, Mistral and Anthropic.
```

File: docs/changelog/5_x.rst (hunk 4 above)
Comment (beerose) [inline docs/changelog/5_x.rst:461]: For consistency — other items don't have a `.` at the end

```suggestion
* Make escaping strings more consistent
  (:eql:gh:`#7059`)

* Separate triggers by kind in compilation
  (:eql:gh:`#7082`)
```

Comment (vpetrovykh) [reply docs/changelog/5_x.rst]: I've removed `.` from these logs, but we're not super-consistent about it overall in changelogs, it seems. Maybe we need to make a decision which is preferred with or without `.` and then do a quick update of all the changelogs?

**Pull request merged**